include/s3select_functions.h

```diff
@@ -769,6 +769,10 @@ struct _fn_to_int : public base_function
       var_result = static_cast<int64_t>(v.dbl());
       break;
 
+    case value::value_En_t::S3NULL:
+      var_result = "null";
```

Comment:
this is not correct.

NULL is not a string, it is a specific value 
the value object contain NULL (`value::value_En_t::S3NULL`)

upon publishing the results (for printout) 
the `to_string` handle the NULL value.